content/docs/configuration/content.md

```diff
@@ -6,58 +6,214 @@ title = "Configuration"
 
 [sidebar]
   enable = true
+  sticky = true
 +++
 
-**Redirect all requests to "www"-subdomain:**  
-*example.com -> www.example.com*
+# Redirect Types
+
+---
+
+## Supported redirect types:
+
+| Type Name | Type Description                                            |
+| --------- | ----------------------------------------------------------- |
+| www       | Redirect all requests to "www"-subdomain                    |
+| host      | Redirect to host provided in TXT record                     |
+| path      | Enable path based redirects                                 |
+| gometa    | Enable Go packages meta/vanity redirects                    |
+| gomods    | Enable serving and caching Go modules                       |
```

Comment:
We'll have to think about the inclusion of this within the docs update or, if it would be better to leave this out.